apps/web/pages/api/auth/[...nextauth].tsx

```diff
@@ -201,6 +202,61 @@ if (isSAMLLoginEnabled) {
       clientSecret: "dummy",
     },
   });
+
+  // Idp initiated login
+  providers.push(
+    CredentialsProvider({
+      id: "saml-idp",
+      name: "IdP Login",
+      credentials: {
+        code: {},
+      },
+      async authorize(credentials) {
+        if (!credentials) {
+          return null;
+        }
+
+        const { code } = credentials;
+
+        if (!code) {
+          return null;
+        }
+
+        const { oauthController } = await jackson();
+
+        // Fetch access token
+        const { access_token } = await oauthController.token({
+          code,
+          grant_type: "authorization_code",
+          redirect_uri: `${process.env.NEXTAUTH_URL}`,
+          client_id: "dummy",
```

Comment:
Should this be configurable or not be dummy?

Reply:
We can leave it as `dummy` or configure `CLIENT_SECRET_VERIFIER`. [More info](https://boxyhq.com/docs/jackson/deploy/env-variables#client_secret_verifier)

Reply:
We should go for CLIENT_SECRET_VERIFIER in .env, if that doesn't exists then switch to default dummy.